desktop/apps/categories3/components/GeneFamilies.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import GeneFamily from './GeneFamily'
+
+const propTypes = {
+  geneFamilies: PropTypes.array.isRequired
+}
```

Comment:
I like this approach to putting `propTypes` at the top so that the interface is immediately clear 👍